jest/jestSetup.js

```diff
@@ -18,6 +20,11 @@ import mockAsyncStorage from '@react-native-community/async-storage/jest/async-s
 polyfillGlobal('URL', () => URL);
 polyfillGlobal('URLSearchParams', () => URLSearchParams);
 
+/**
+ * Default should be set with `timers: 'modern'` in Jest config.
+ */
+assertUsingModernFakeTimers();
```

Comment:
nit: this isn't a declaration of anything, so there's nothing the comment is describing the interface of -- so it should be a plain non-jsdoc implementation-comment